docs/post/controllers.md

```diff
@@ -101,17 +102,29 @@ This allows you to take advantage DualShock 3's pressure sensitive face button o
 4. Enable the XInput source in PCSX2.
 5. Use "Automatic Binding" button and select XInput.
 
-### Linux and Mac (Experimental)
+## Custom profile
 
-The auto binding system doesn't fully support it, but the backend does.
+### What is the purpose of a controller profile?
 
-- Bind all your keys, then close PCSX2 and edit the `PCSX2.ini` file to delete all the buttons and change all the axes from `-Axis##` or `+Axis##` to `FullAxis##`
+Custom profiles are intended for overriding your normal Shared mappings. You use profiles when you want a completely different controller setup for a specific game. For example, you may create a profile for games which use a guitar controller, rather than changing your mappings every time you play that one game. If you aren't trying to make a special setup for one specific game, don't use profiles.
 
-:::caution
-On Linux, [you may need to add udev rules to give PCSX2 access to the controller](https://wiki.rpcs3.net/index.php?title=Help:Controller_Configuration#On_Linux).
-:::
+### Why isn't my game recognizing my profile?
+
+The Shared profile is what you start out with and is what all games default to, unless told otherwise. When creating a profile, the name prompt tells you to change the input profile in your per-game settings.
+
+If you want to use a custom profile for a specific game, then you need to assign it to the game. You can find this option on the per-game setting's summary tab, Right click it in your games list, hit Properties, then select the appropriate item under Input Profile.
```

Comment:
Capitalize Summary to match the capitalization as in the UI. I would also switch the comma after "Summary tab" to a period.